src/slim-select/select.ts

```diff
@@ -107,6 +107,19 @@ export default class Select {
         if (m.attributeName === 'class') {
           classChanged = true
         }
+
```

Comment:
Can you describe what you are doing here?

Reply:
A new option was added that is now the selected value, therefore we trigger a `change` event, which does not fire normally without user interaction. This fixes one of the old tests, which failed because the event was not fired before.  I'm pretty sure that this is the way you expect it to work, but let me know